src/main/java/umk/mat/pajda/ProjektZespolowy/configs/SecurityConfig.java

```diff
@@ -0,0 +1,57 @@
+package umk.mat.pajda.ProjektZespolowy.configs;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+public class SecurityConfig {
+
+  @Bean
+  public UserDetailsService userDetailsService() {
+    UserDetails user =
+        User.builder()
+            .username("user")
+            .password(passwordEncoder().encode("user"))
```

Comment:
shouldn't this be more random string hidden in env variables? 🤔

Reply:
i added it: https://github.com/Programowanie-Zespolowe-PAJDA/Backend/pull/15